.tekton/lvm-operator-pull-request.yaml

```diff
@@ -82,11 +84,11 @@ spec:
       description: Skip checks against built image
       name: skip-checks
       type: string
-    - default: "false"
+    - default: "true"
       description: Execute the build with network isolation
       name: hermetic
       type: string
-    - default: ""
+    - default: '{"packages": [{"type": "gomod"}], "flags": ["gomod-vendor-check"]}'
```

Comment:
I see in the job logs that `gomod-vendor-check` is deprecated and is no longer needed (enabled by default). Shall we remove it?